src/tree_map.h

```diff
@@ -57,6 +57,19 @@ enum TreeGround {
 	TREE_GROUND_ROUGH_SNOW  = 4, ///< A snow tile that is rough underneath.
 };
 
+/**
+ * Enumeration for tree growth stages.
+ *
+ * This enumeration defines the stages of tree growth for tiles with trees on it.
+ */
+enum class TreeGrowthStage : uint {
+	Growing1 = 0,  ///< First stage of growth
+	Growing2 = 1,  ///< Second stage of growth
+	Growing3 = 2,  ///< Third stage of growth
+	Grown = 3,  ///< Fully grown tree
+	Dying1 = 5,  ///< First stage of dying
```

Comment:
Does the integral value 4 have any meaning in this?

Reply:
Oops :) For some reason I thought there are only 2 dying stages

Reply:
Actually, the last one is kinda fully dead so might as well name it as such I guess